### DISCUSSION TITLE
Programmazione mensile shelly plus

### DISCUSSION THREAD

**Discussion by alessiomanzi** [indomus/forum#2385] [GENERALE - Discussioni generiche]
Salve a tutti, spiego il mio problema. Ho dei condizionatori comandati da uno Shelly plus PM con Add On e sonda, oltre che al sensore porta Shelly Door 2. All'apertura del contatto porta ho impostato lo stop del PM così come d'estate al raggiungimento di una soglia di es.: 25 °C. In inverno però chiaramente la funzione si inverte e vorrei trovare un modo semplice senza passare per HA che mi inverta l'impostazione, e senza ovviamente, riprogrammare tutti i PM (ne sono un bel pò) . Mi hanno consigliato degli script, ma non ho molta esperienza sul codice.

**Comment by snakuzzo** [comment]
Guarda [QUI](https://shelly-api-docs.shelly.cloud/gen2/Scripts/Tutorial/)

**Reply by alessiomanzi** [reply]
Grazie per la risposta, ma purtroppo come ho specificato nel post , non sono molto pratico riguardo al codice. Saresti così gentile da farmi un esempio? Così comincio a fare un pò di pratica...(no problem about a coffee). Grazie. Alessio
